spec/api-menu-spec.js

```diff
@@ -288,6 +288,37 @@ describe('Menu module', () => {
     })
   })
 
+  describe('Menu.closePopup()', () => {
+    let w = null
+    let menu
+
+    beforeEach((done) => {
+      w = new BrowserWindow({show: false, width: 200, height: 200})
+      menu = Menu.buildFromTemplate([
+        {
+          label: '1'
+        }
+      ])
+
+      w.loadURL('data:text/html,<html>teszt</html>')
+      w.webContents.on('dom-ready', () => {
+        done()
+      })
+    })
+
+    afterEach(() => {
+      return closeWindow(w).then(() => { w = null })
+    })
+
+    it('emits closed event', (done) => {
+      menu.popup(w, {x: 100, y: 100, async: true})
```

Comment:
`async` option is removed with chromium 61. Also worth removing the other reference in this file that was missed out in https://github.com/electron/electron/commit/4e859b47185cb441b210e4bd85c10644088e4106#diff-0f899d7673156733456b3d561fe6c8fe

Reply:
Done!